src/builder/QuotecallWrapper.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import {CodeJarHelper} from "./CodeJarHelper.sol";
+import {IQuarkWallet} from "quark-core/src/interfaces/IQuarkWallet.sol";
+import {Quotecall} from "../Quotecall.sol";
+
+// Helper library to wrap a QuarkOperation from Actions.sol for a Paycall
+library QuotecallWrapper {
+    function wrap(IQuarkWallet.QuarkOperation memory operation, uint256 chainId, uint256 quotedAmount)
+        internal
+        pure
+        returns (IQuarkWallet.QuarkOperation memory)
+    {
+        bytes[] memory scriptSources = new bytes[](1);
+        scriptSources[0] = type(Quotecall).creationCode;
```

Comment:
Same comment as in #12 about script sources.

Reply:
👍 Change this PR#14 to rebase on top of #12 which now has the information needed to calculate the address for paycall/quotecall.